### PR TITLE
Update website for 2020-12 patch release

### PR DESCRIPTION
**Pull request by Relequestual**
As required for the release process.
See https://github.com/json-schema-org/json-schema-spec/issues/1191

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -64,6 +64,10 @@ We monitor the `jsonschema` tag on StackOverflow.
 
 ## Project Status
 
+2021-06-10: A patch release of Draft 2020-12 has been published with no functional changes.
```
Comment (Relequestual) [inline index.md:67]:
```suggestion
2022-06-10: A patch release of Draft 2020-12 has been published with no functional changes.
```

**Review by Julian — APPROVED** [review]
lgtm

File: .gitmodules [hunk 2]
```diff
@@ -10,15 +10,18 @@
 [submodule "_includes/draft-03"]
 	path = _includes/draft-03
 	url = https://github.com/json-schema-org/json-schema-spec.git
+		branch = draft-03
```
Comment (jviotti) [inline .gitmodules:13]: Indentation seems wrong on these `branch` attributes

Comment (Relequestual) [reply .gitmodules:13]: Thanks, I'll fix that.

**Comment by Relequestual** [comment]
With no objections and no further reviews, I'll merge.
If there are any issues, we can fix as we see them.

**Pull request merged**